controllers/mysqlserver_controller.go

```diff
@@ -68,7 +68,7 @@ func makeMapFunc(mgr ctrl.Manager) func(o client.Object) []reconcile.Request {
 		// This should be fast since the list of items it's going through should always be cached
 		// locally with the shared informer.
 		// Unfortunately we don't have a ctx we can use here... maybe need to file a bug?
```

Comment:
Do file an issue and give the # here.

Reply:
Filed - although when I had talked with the controller-runtime folks before they weren't super optimistic about being able to do anything about this...